langchain/src/tools/apify.ts

```diff
@@ -0,0 +1,89 @@
+import { Document } from "../document.js";
+import { ApifyDatasetLoader } from "../document_loaders/web/apify_dataset.js";
+
+export interface ApifyCallActorOptions {
+  /**
+   * Tag or number of the actor build to run (e.g. `beta` or `1.2.345`).
+   * If not provided, the run uses build tag or number from the default actor run configuration (typically `latest`).
+   */
+  build?: string;
+
+  /**
+   * Memory in megabytes which will be allocated for the new actor run.
+   * If not provided, the run uses memory of the default actor run configuration.
+   */
+  memory?: number;
```

Comment:
Let's use units everywhere (and later fix it also in `ApifyClient` to accept both the correct `memoryMbytes` and the wrong `memory`).
```suggestion
  memoryMbytes?: number;
```